src/commands/settings.ts

```diff
@@ -9,6 +9,8 @@ const editableOptions = [
 	'channels.nsfw-logs',
 	'channels.event-logs',
 	'channels.event-logs.blacklist',
+	'channels.matchmaking',
+	'matchmaking.lock-timeout-seconds',
```

Comment:
Not your fault since it was like this before, but we prefer to not have trailing commas